app/views/admin/loan_questions/_form.slim

```diff
@@ -16,9 +16,21 @@
 
     = f.input :data_type, as: :select, label: t('loan_questions.data_type'),
       collection: @data_types
-      
+
```

Comment:
If you see it, let me know. Atom has issues with Slim files, so though I have settings to see it when it happens, sometimes I miss it. Thanks and sorry about that.